packages/ubuntu_wizard/lib/src/l10n/ubuntu_pt_BR.arb

```diff
@@ -1,5 +1,6 @@
 {
   "@@locale": "pt_BR",
+  "countryCode": "BR",
```

Comment:
Technically, the country code is not necessary for .arb files that already specify a full locale. I specified it here regardless, to make it consistent with others.